# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ptouch-driver/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -urp ptouch-driver-1.2.orig/rastertoptch.c ptouch-driver-1.2/rastertoptch.c
--- ptouch-driver-1.2.orig/rastertoptch.c	2006-09-08 16:39:27.000000000 +0200
+++ ptouch-driver-1.2/rastertoptch.c	2008-02-13 16:37:13.000000000 +0100
@@ -311,6 +311,7 @@
 #include <math.h>
 #include <cups/raster.h>
 #include <cups/cups.h>
+#include <limits.h>
 
 #if STDC_HEADERS
 # include <string.h>

